modules/plugins/languages/haskell.nix

```
@@ -0,0 +1,99 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}: let
+  inherit (builtins) isList;
+  inherit (lib) options modules nvim strings types;
+  inherit (types) either package listOf str;
+  inherit (options) mkEnableOption mkOption;
+  inherit (strings) optionalString;
+  inherit (modules) mkIf mkMerge;
+  inherit (nvim.types) mkGrammarOption;
+  inherit (nvim.dag) entryAnywhere;
+  inherit (nvim.lua) expToLua;
```

```suggestion
  inherit (lib.modules) mkIf mkMerge;
  inherit (lib.options) mkEnableOption mkOption;
  inherit (lib.strings) optionalString;
  inherit (lib.lists) isList;
  inherit (lib.types) either package listOf str;
  inherit (lib.nvim.types) mkGrammarOption;
  inherit (lib.nvim.dag) entryAnywhere;
  inherit (lib.nvim.lua) expToLua;
```

Please remain consistent with the project's style.

---

It is now consistent.